src/components/NotificationMessage/NotificationMessage.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-lone-blocks */
+import React from 'react';
+import cn from 'classnames';
+
+import { Notification } from '../../types/Notification';
+
+type Props = {
+  data: Notification;
+};
+
+export const NotificationMessage: React.FC<Props> = ({
+  data: {
+    typeNotification,
+    text,
+    dataCypress,
+  },
+}) => (
+  <div
+    className={cn(
+      'notification',
+      {
+        'is-warning': typeNotification === 'warning',
+        'is-danger': typeNotification === 'danger',
```

Comment:
You can create an enum for notification types to avoid errors connected with typos